common/models/user.js

```diff
@@ -42,11 +42,15 @@ var debug = require('debug')('loopback:user');
  * - ALLOW OWNER `findById`
  * - ALLOW OWNER `updateAttributes`
  *
- * @property {String} username Must be unique
- * @property {String} password Hidden from remote clients
- * @property {String} email Must be valid email
- * @property {Boolean} emailVerified Set when a user's email has been verified via `confirm()`
- * @property {String} verificationToken Set when `verify()` is called
+ * @property {String} username Must be unique.
+ * @property {String} password Hidden from remote clients.
+ * @property {String} email Must be valid email.
+ * @property {Boolean} emailVerified Set when a user's email has been verified via `confirm()`.
+ * @property {String} verificationToken Set when `verify()` is called.
+ * @property {String} realm Specific domain a user belongs to, [Learn more](https://docs.strongloop.com/display/public/LB/Partitioning+users+with+realms).
+ * @property {Date} created Unused properties (Deprecated).
```

Comment:
I am proposing a slightly different text (please apply it to all three unused properties):

```
* @property {Date} created The property is not used by LoopBack, you are free to use it for your own purposes. 
```

On the second thought, I think it's not necessary to mention that we are thinking about removing this property in the future. IMO it's enough to mention the removal once it's done, because the upgrade path would be pretty straightforward (define these three properties in your subclassed User model).

@crandmck please review